backend/OR-Tools/OR-Tools/Program.cs

```diff
@@ -7,6 +7,7 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+            while (true) { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4594462)